everest-testing/src/everest/testing/core_utils/test_control_module.py

```diff
@@ -0,0 +1,114 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+
+import logging
+import os
+import time
+from pathlib import Path
+import importlib
+import sys
+from os import environ
+
+
+class TestControlModule:
+    """This class can be used to configure, start and stop the EVerest PyTestControlModule
+    """
+
+    def __init__(self, everest_core_path: Path) -> None:
+        """Initialization method for TestControlModule
+        """
+        self.process = None
+        self.current_file_path = Path(__file__).absolute()
+        self.everest_core_path = everest_core_path
+        self.workspace_path = everest_core_path.parent
+        self.everest_core_build_path = self.everest_core_path / 'build'
+        self.everest_core_build_dist_path = self.everest_core_build_path / 'dist'
+        self.everest_core_module_path = self.everest_core_build_dist_path / 'libexec/everest/modules'
+        self.test_control_module_bin_path = self.everest_core_module_path / 'PyTestControlModule/module.py'
+        self.everest_py_path =  self.everest_core_build_dist_path / 'lib/everest/everestpy/everest.py'
+        self.everest_core_config_path = self.everest_core_path / 'config'
```

Comment:
this is quite error prone, all the constants should be moved to a reusable place, other test modules will require this as well I guess